content/docs/06-integrations/00-byoos.md

```diff
@@ -0,0 +1,239 @@
+---
+title: "Bring Your Own OS (BYOOS)"
+metaTitle: "Bring your own OS (BYOOS)"
+metaDescription: "Learn how to use a custom Operating System with a cluster profile."
+hiddenFromNav: true
+type: "integration"
+category: ['operating system']
+logoUrl: "https://registry.dev.spectrocloud.com/v1/spectro-proxy/blobs/sha256:b6081bca439eeb01a8d43b3cb6895df4c088f80af978856ddc0da568e5c09365?type=image/png"
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+
+# Bring Your Own Operating System (BYOOS) 
+
+The [Bring Your Own Operating System (BYOOS)](/https://docs.spectrocloud.com/cluster-profiles/byoos) enables the ability for you to use a custom Operating System (OS) with Palette. Palette comes with several OS out-of-the-box, but for some users, the existing list of OS may not meet their needs. 
+
+Using your custom OS provides several benefits, including the ability to control your own dependencies, improve performance, and ensure compatibility with your existing applications. With BYOOS, you can choose the OS that best fits your needs, whether it's a commercial or open-source distribution, and integrate it with your Kubernetes clusters. The BYOOS pack can be used with both Edge and non-Edge environments. 
```

Comment:
@karl-cardenas-coding From an earlier comment/question in this PR - so it sounds customers choose the OS they want to use. They don't create it. Earlier I think I read they create it, which doesn't sound accurate.

Maybe users can customize an OS of their choice?